src/services/refactors/inlineVariable.ts

```diff
@@ -115,7 +120,12 @@ registerRefactor(refactorName, {
         const { references, declaration, replacement } = info;
         const edits = textChanges.ChangeTracker.with(context, tracker => {
             for (const node of references) {
-                tracker.replaceNode(file, node, getReplacementExpression(node, replacement));
+                if (isStringLiteral(replacement) && isIdentifier(node) && isTemplateSpan(node.parent) && !isTaggedTemplateExpression(node.parent.parent.parent)) {

```

Comment:
I suspect `isTaggedTemplateExpression(node.parent.parent.parent)` isn't sufficient due to parens; they may need to be unwrapped / walked up (we have a helper for it).

Reply:
Did you mean ``$(`echo ${message}`);`` ? But they are parsed as call expression.

Reply:
I mean `` $`echo ${(message)}` ``